TESTING-INSTRUCTIONS.md

```diff
@@ -26,6 +26,113 @@ add_filter( 'woocommerce_admin_export_id', function ($export_id) {
 } );
 ```
 2. Repeat the same steps from above. The filename in the link now should be `different_export_id`.
+### Payment gateway suggestions feature
+
+1. Navigate to the homescreen via WooCommerce -> Home
+2. Click on "Set up payments"
+
+#### How to test
+
+Individual payment gateway plugins dictate the settings and connection flow.  For testing purposes, we'll test both the default behavior of the gateway and the enhanced configuration behavior.
+
+1. On the payments task, click "Set up" or "Enable" next to a gateway
+2. Note that the gateway is installed if it requires a plugin
+3. On the connection step, a button should be presented that links to the gateway's (legacy) settings screen
+4. Delete the plugin
+5. Install some of the payment gateways from the links below.  They don't need to be activated, but the folder names should match those in the zip.
+6. If setting fields are shown, make sure that validation works, input is saved, and is persisted on page refresh.  Make sure the gateway is marked as enabled and not labeled "Requires setup" if all fields are completed.
+7. If the "Connect" button is shown, follow the connection flow.  Make sure that you are returned to the payments task and that the gateway is enabled and marked as configured.
+8. Remove some settings manually under the payment gateway's legacy settings screen.  Make sure the gateway is no longer marked as configured.
+
+* Klarna - https://mc.a8c.com/includes/img-uploader/files/1624475036-klarna-checkout-for-woocommerce.zip
+* PayFast - https://mc.a8c.com/includes/img-uploader/files/1624475037-woocommerce-gateway-payfast.zip
+* PayPal - https://mc.a8c.com/includes/img-uploader/files/1624475037-woocommerce-paypal-payments.zip
+* RazorPay - https://mc.a8c.com/includes/img-uploader/files/1624475037-razorpay-woocommerce.zip
+* Stripe - https://mc.a8c.com/includes/img-uploader/files/1624475040-woocommerce-gateway-stripe.zip
```

Comment:
@joshuatf this is not a huge issue, but I noticed that this stripe zip version contains this as the setupHelpText:
```
PayU can be configured under your store settings. Create your PayU account here.
```
The PR for the stripe plugin looks correct.